pkg/volume/util/volumepathhandler/volume_path_handler.go

```diff
@@ -112,29 +112,32 @@ func mapBindMountDevice(v VolumePathHandler, devicePath string, mapPath string,
 	// Check bind mount exists
 	linkPath := filepath.Join(mapPath, string(linkName))
 
-	file, err := os.Stat(linkPath)
-	if err != nil {
-		if !os.IsNotExist(err) {
-			return fmt.Errorf("failed to stat file %s: %v", linkPath, err)
-		}
+	file, err := os.Lstat(linkPath)
 
-		// Create file
-		newFile, err := os.OpenFile(linkPath, os.O_CREATE|os.O_RDWR, 0750)
-		if err != nil {
-			return fmt.Errorf("failed to open file %s: %v", linkPath, err)
-		}
-		if err := newFile.Close(); err != nil {
-			return fmt.Errorf("failed to close file %s: %v", linkPath, err)
-		}
-	} else {
+	if err == nil {
 		// Check if device file
 		// TODO: Need to check if this device file is actually the expected bind mount
 		if file.Mode()&os.ModeDevice == os.ModeDevice {
 			klog.Warningf("Warning: Map skipped because bind mount already exist on the path: %v", linkPath)
 			return nil
 		}
```

Comment:
It is no need check whether it is symlink or not, just kept what we want and remove others. 
This can keep the code simple and clean.

Reply:
It may be simple, but not clean.
Because, it's not clear why the path needs to be deleted and there will be a case that the path is unnecessarily deleted.